packages/odyssey-react-mui/src/i18n.ts

```diff
@@ -47,7 +47,7 @@ export const defaultLNG = "en";
 export const defaultNS = "translations";
 
 // Note: This is type "string" to allow translation overrides from other languages
-export type I18nResources = Record<string, typeof en>;
+export type I18nResources = Record<string, Partial<typeof en>>;
```

Comment:
add comment @KevinGhadyani-Okta

Reply:
For visibility, see: https://okta.slack.com/archives/C7T2H3KNJ/p1702418981364249?thread_ts=1702053051.736909&cid=C7T2H3KNJ

Reply:
We can remove the `Partial` now (as you said).